src/Ledger/BaseTypes.lagda

```diff
@@ -21,3 +23,10 @@ private
 \end{code}
 \caption{Some basic types used in many places in the ledger}
 \end{figure*}
+
+\begin{figure*}[h]
+\begin{code}
+  UnitInterval = [ x ∈ ℚ ∣ (0ℚ ≤ x) × (x ≤ 1ℚ) ]
```

Comment:
It's probably a good idea to use literal overloading, so you can just write `0` and `1`. To do that, you have to add the `Number` instance from `Data.Rational.Literals`, ideally to the `Prelude` module next to the instance for `ℕ`. Then, just add `open import Agda.Builtin.FromNat` to any module that should use those overloaded literals.

Reply:
I prefer `0` and `1` as well, I will give it a try.